coalib/bears/Bear.py

```diff
@@ -16,14 +16,31 @@
 from coalib.bears.BEAR_KIND import BEAR_KIND
 from coalib.output.printers.LogPrinter import LogPrinterMixin
 from coalib.results.Result import Result
+from coalib.misc.Constants import FALSE_STRINGS, TRUE_STRINGS
 from coalib.results.TextPosition import ZeroOffsetError
 from coalib.settings.FunctionMetadata import FunctionMetadata
 from coalib.settings.Section import Section
 from coalib.settings.ConfigurationGathering import get_config_directory
+from coalib.settings.Setting import typed_list
 
 from .meta import bearclass
 
 
+def _debugger_helper(bear):
```

Comment:
what does this do?

the function name should describe it.

the docstring should explain it.